.github/workflows/ruby.yml

```diff
@@ -32,3 +32,8 @@ jobs:
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
     - name: Run tests
       run: bundle exec rake
+    - name: Run tests for DAP
```

Comment:
This adds another step to all testing jobs. What you want to do may be adding `workflows/dap.yml`